configs/tsconfig.json

```diff
@@ -4,6 +4,7 @@
     "jsx": "react-jsx",
     "jsxImportSource": "@emotion/react",
     "target": "ES2019",
+    "esModuleInterop": true,
```

Comment:
This was probably not it (it was still necessary to update @emotion/react first), but it does not seem to hurt either.